style.css

```diff
@@ -0,0 +1,1547 @@
+  }
+
+  .resume {
+    background-color: #ff6b00;
+    color: white;
+    border: none;
+    padding: 16px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 20px 25px;
+    cursor: pointer;
+    list-style: none;
+    margin-left: 80px;
+  }
+
+  .resume:hover {
+    background-color: antiquewhite;
+    color: #ff6b00;
+  }
+
+  .line2 {
+    width: 80%;
+    border: 1px solid gray;
+    height: 1px;
+    margin-left: 100px;
+    margin-right: 142px;
+    margin-top: 150px;
+  }
+
+  .new-vector {
+    display: none;
+  }
+
+  .information {
+    color: white;
+    display: flex;
+    flex-direction: row;
+
+    /* align-items: center; */
+    justify-content: center;
+    list-style: none;
+    text-align: center;
+    gap: 150px;
+    margin-top: 146px;
+    width: 100%;
+    padding-right: 170px;
+    padding-left: 170px;
+  }
+
+  .information-list {
+    list-style: none;
+    font-family: "Inter", serif;
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 44px;
+    text-align: left;
+    margin-left: -50px;
+  }
+
+  .list {
+    list-style: none;
+    font-family: "Inter", serif;
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 43px;
+    text-align: left;
+  }
+
+  #contact-form {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 100%;
+    height: 590px;
+    gap: 20px;
+    padding: 30px;
+  }
+
+  .contact-text {
+    width: 455px;
+
+    /* max-width: fit-content; */
+    height: 256px;
+    font-family: "Inter", serif;
+
+    /* font-style: normal; */
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 44px;
+
+    /* text-align: center; */
+    color: #262626;
+    margin-top: 130px;
+
+    /* margin-left: 20px; */
+
+    /* position: absolute; */
+    padding-left: 16px;
+  }
+
+  .form {
+    display: none;
+  }
+
+  .form2 {
+    display: flex;
+
+    /* width: 80%; */
+
+    /* margin-left: 730px; */
+    margin-top: 130px;
+    width: 100%;
+  }
+
+  .name-sur {
+    display: flex;
+    flex-direction: row;
+    gap: 15px;
+  }
+
+  input {
+    width: 100%;
+    margin: 10px 0;
+    border: none;
+    border-bottom: 1px solid #dbd8d7;
+    height: 30px;
+  }
+
+  input::placeholder {
+    font-family: "Inter", serif;
+    font-size: 15px;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    padding-left: 12px;
+  }
+
+  /* textarea {
+    width: 130%;
+    height: 114px;
+    margin-top: 15px;
+    border: none;
+    background-color: #fbf8f7;
+    background-image: url("icons/text-area-icon.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    resize: none;
+  }
+
+  textarea::placeholder {
+    font-family: "Inter", serif;
+    font-size: 15px;
+    font-style: italic;
+    font-weight: 400;
+    line-height: 24px;
+    text-align: left;
+    height: 90px;
+    padding-left: 12px;
+    padding-top: 12px;
+    color: #3c3a39;
+  } */
```

Comment:
- [ ] For clean code and best practices reasons, kindly make sure you remove all commented and unused code lines from your code base